### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by joshwlambert**
The readme is updated to include information on contributing to the spreadsheet of data available on the google sheet. Updates also include updating plotting, adding code of conduct, contributing to the package guidelines to readme.

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -48,6 +48,40 @@ incubation_H7N9 <- epiparameter::epidist(
...
+
+## Contributing to library of delay distributions
+
+If you would like to contribute to the different delay distributions stored in the `epiparameter` package, you can access the [google sheet](https://docs.google.com/spreadsheets/d/1VzC8NFHSQYPKsiDnxA7gDsm4cVr79outKQZJP8cC7Ro/edit?usp=sharing) and add your data.
+
+Notes on the the spreadsheet:
+
+- Most studies will report the distribution(s) they fitted to the data. In the case that the study did not report the distribution of the reported parameters or summary statistics (mean and standard deviation of the distribution) the distribution can be assumed (e.g. by looking at the shape of the distribution from figures in the paper). When a distribution is being assumed and not explicitly stated please add this assumption to the notes section of the spreadsheet.  We are only interested in studies that report the distribution fitted to the data. This will ensure that if this data is used in another application the user can be alerted that the distribution is assumed and not known with certainty.
```
Comment (pratikunterwegs) [inline README.Rmd:65]: Perhaps rephrase to "We are currently only accepting studies that report the distribution ..."

**Review by pratikunterwegs — APPROVED** [review]
Thanks @joshwlambert, looks alright to me. There are linter warnings on `vignettes/epiparameter.Rmd`, but otherwise it looks fine. I only have one minor suggestion that you can ignore if you really want to discourage people from adding anything to the spreadsheet other than distributions fitted to real data (e.g. are there alternatives from simulations that are reliable?).

File: README.Rmd (hunk 1 above)
Comment (joshwlambert) [reply README.Rmd:65]: The vignette said something similar to what you suggested in a previous version but we decided to updated to give more freedom to assumed distributions. https://github.com/epiverse-trace/epiparameter/blob/d104b0fdb42fd53cc360acfcadd2f3ebeef872bd/vignettes/epiparameter.Rmd#L455

**Comment by joshwlambert** [comment]
Thanks for the comments. I'm going to leave the lint warnings for now as it is just about camelCase variables.

**Pull request merged**